packages/calcite-components/src/components/tree/tree.tsx

```diff
@@ -175,7 +175,8 @@ export class Tree {
       (((this.selectionMode === "single" || this.selectionMode === "multiple") &&
         childItems.length <= 0) ||
         this.selectionMode === "children" ||
-        this.selectionMode === "multichildren");
+        this.selectionMode === "multichildren" ||
+        this.selectionMode === "single-persist");
```

Comment:
Should we add an e2e test to account for single-persist allowing to be unselected?

Reply:
Done.